src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
```

Comment:
According to the encapsulation principle, it's a bad practice to make your class elements exposed. Consider using private access modifier for the 'fruits' map and provide public methods to manipulate the data if necessary.